test/soft_delete_repo_test.exs

```diff
@@ -136,5 +136,27 @@ defmodule Ecto.SoftDelete.Repo.Test do
 
       assert length(results) == 1
     end
+
+    test "returns same result for different types of where clauses", %{topic: topic} do
```

Comment:
```suggestion
    test "returns same result for different types of where clauses" do
```
the second argument `%{topic: topic}` is causing the test to fail.

Reply:
Oh, thank you for pointing it out. 
Was checking something out before pushing my changes and forgot to remove this. :sweat_smile: